src/main/resources/db/migration/V6__Add_overall_league_table.sql

```diff
@@ -0,0 +1,12 @@
+create table overall_league
+(
+    zookid   SMALLSERIAL      not null,
+    name     VARCHAR(25)      null,
```

Comment:
```suggestion
    name     VARCHAR(25)      not null,
```
zooks should always have names